packages/devtools-reps/src/object-inspector/component.js

```diff
@@ -289,12 +332,30 @@ class ObjectInspector extends Component<Props> {
         repProps.mode = MODE.TINY;
       }
 
-      if (nodeIsLongString(item)) {
+      if (
+        nodeIsLongString(item) ||
```

Comment:
A more general question, is there any way for evaluations to be treated like any other item? I suppose this might mean that we need to do something with item and values, but we are already doing work with children...